### PR TITLE
[CCTRI-2663] - alpine & python version update

### PR DESCRIPTION
**Pull request by ytsek**
- alpine version update
- python version update

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
-FROM alpine:3.13
+FROM alpine:3.14
 LABEL maintainer="Ian Redden <iaredden@cisco.com>"
 
 # install packages we need
-RUN apk update && apk add --no-cache musl-dev openssl-dev gcc python3 py3-configobj python3-dev supervisor git libffi-dev uwsgi-python3 uwsgi-http jq nano syslog-ng uwsgi-syslog py3-pip
+RUN apk update && apk add --no-cache musl-dev openssl-dev gcc py3-configobj \
+supervisor git libffi-dev uwsgi-python3 uwsgi-http jq syslog-ng uwsgi-syslog \
```
Comment (oshynk) [inline Dockerfile:6]: Why do we need git inside a container?

Comment (oshynk) [reply Dockerfile:6]: And jq

Comment (ytsek) [reply Dockerfile:6]: We use git in [entrypoint.sh](https://github.com/CiscoSecurity/tr-05-serverless-google-safe-browsing/blob/develop/scripts/entrypoint.sh)
And jq in [start.sh](https://github.com/CiscoSecurity/tr-05-serverless-google-safe-browsing/blob/develop/scripts/start.sh)

Comment (ytsek) [reply Dockerfile:6]: Yes, you are totally right, the are unnecessary inside a container. I would remove them.

**Review by oshynk — APPROVED** [review]

**Review by MaxMhonc — APPROVED** [review]

**Pull request merged**